examples/widget.rs

```diff
@@ -0,0 +1,61 @@
+use std::path::PathBuf;
+use ratatui::{
+    crossterm::event::{KeyCode},
+    Frame,
+};
+use binsider::prelude::Event;
```

Comment:
Can you remove the other implicit imports (e.g. `binsider::file` and so on) and use the prelude module? Feel free to update the prelude if we are not already covering some of the types.

Reply:
done